src/locales/en/translation.json

```
@@ -164,14 +163,35 @@
       "data_unavailable": "Project data is currently unavailable.",
       "duplicate_name": "A project with the same name already exists.",
       "not_saved": "Project could not be saved",
-      "not_available": "Projects are currently unavailable."
+      "not_available": "Projects are currently unavailable.",
+      "not_deleted": "This project has not been deleted."
     },
     "success": {
       "project_copied": "Project copied",
       "project_created": "Project created",
       "project_saved": "Project saved",
+      "project_deleted": "Project deleted",
       "test_project_status_saved": "Test project selection saved."
-    }
+    },
+    "project_info": "Project info",
+    "no_notes": "No notes for this Project",
+    "no_organizations": "This Project has no organizations.",
+    "project_admin_one": "Project Admin",
+    "project_admin_other": "Project Admins",
+    "other_project_member_one": "Other Project Member",
+    "other_project_member_other": "Other Project Members",
+    "project_last_updated": "Project Last Updated",
+    "country_one": "Country",
+    "country_other": "Countries"
+  },
+  "delete_project_modal": {
+    "title": "Delete project",
+    "prompt": "Are you sure you want to delete {{projectName}}?",
+    "yes": "Delete {{projectName}}",
+    "has_sample_units": "You must delete all the sample units in this project to delete this project.",
+    "has_other_users": "Other users must be removed from this project before deletion.",
+    "cannot_delete": "You cannot delete {{projectName}} because it is used in the following sample units:",
+    "remove_before_delete": "You have to remove {{projectName}} from all sample units before you can delete it."
```

Could we move these to the 'project' namespace?
```suggestion
  "project": {
    "delete": "Delete project",
    "user_confirm_delete": "Are you sure you want to delete {{projectName}}?",
    "confirm_delete": "Delete {{projectName}}",
    "delete_sample_units_notice": "You must delete all the sample units in this project to delete this project.",
    "delete_other_users_notice": "Other users must be removed from this project before deletion.",
    "cannot_delete": "You cannot delete {{projectName}} because it is used in the following sample units:",
    "remove_before_delete": "You have to remove {{projectName}} from all sample units before you can delete it."
```